app_customers/lib/data/database/hive_service.dart

```diff
@@ -452,6 +447,11 @@ class HiveServiceImpl implements HiveService {
     final headers = {
       PreferencesKeys.clientVersion: packageInfo.version,
     };
+
+    /// TODO: updates the url once the backend is ready.
+    /// TODO: updates the url once the backend is ready.
+    /// TODO: updates the url once the backend is ready.
```

Comment:
Please remove all that.

Reply:
This help us to remember some update we need to do here.